lib/adminware/job_list.rb

```diff
@@ -0,0 +1,98 @@
+require 'adminware'
+require 'adminware/config'
+require 'adminware/schedule'
+require 'fileutils'
+require 'terminal-table'
+
+module Adminware
+  module JobList
+    class << self
+      #Handle the input command
+      def run(name, host, boolean)
+        @name = name
+        @host = host
+        @config = Adminware.config
+        @plain = boolean
+        @description = nil
+
+        if @host == 'local' 
+          list_job_values
+        else
+          search_remote_host
+        end
+      end
+      
+      #Lists the values of a job within the state file
+      def list_job_values
+        jobs = []
+        Dir.foreach("#{@config.jobdir}") do |item|
+          next if item == '.' or item == '..'
+          jobs << item 
+        end 
+
+        if jobs.empty?
+          puts "\t> No jobs to list"
+        else
+          which_output?(jobs)
```

Comment:
This isn't a boolean check.  Probably rename to something more declarative, simply `output`?

Reply:
Or `render`?